src/controllers/management/borrowing_department.py

```diff
@@ -13,71 +13,61 @@ class BorrowingDepartment:
     def __init__(self):
         self.controller_manager = ControllersManager()
 
-    def return_item(self, library_item: LibraryItem, patron_id: str):
+    async def return_item(self, library_item: LibraryItem, patron_id: str):
         """
-        Handles the return process of a library item by a patron.
-
-        This function updates the library system when a patron returns a borrowed item. It checks if the patron
-         exists, verifies   if the item is in the library's inventory and currently borrowed, calculates any
-        outstanding bills, and processes the return if all conditions are met.
-
-        Parameters:
-        - library (Library): The Library object representing the library system.
-        - library_item (LibraryItem): The LibraryItem object (book, disk, etc.) being returned.
-        - patron_id (str): The unique identifier for the patron returning the item.
-
-        Raises:
-        - ValueError: If the patron does not exist, the item is not in the library, the item is not borrowed,
-                      or if the patron has outstanding bills.
+        Handles the asynchronous return process of a library item by a patron.
         """
```

Comment:
Why did you delete the doc

Reply:
While i change the code is deleted in mistake - repeat it from main.